### PR TITLE
Add `activity_output`

### PR DESCRIPTION
**Pull request by t-eckert**
## What is the goal of this pull request?

Add a type returned by Activity Functions.

## What does this pull request change?

Creates a new file `azure-functions/src/durable/activity_output.rs` with the ActivityOutput type.

## What work remains to be done?

Add test, docs, flesh out if this is the right approach.

## Do you consider it adequately tested?

NO, I should add some unit tests while we draft this.

### CODE REVIEW

File: azure-functions/src/durable/activity_output.rs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+use serde_json::Value;
+
+/// # Activity Result
+/// 
+/// Type returned by Activity Functions for Durable Functions
+
+struct ActivityResult(serde_json::Value);
```
Comment (peterhuene) [inline azure-functions/src/durable/activity_output.rs]: Should be named `ActivityOutput` :)

Comment (t-eckert) [reply azure-functions/src/durable/activity_output.rs]: D'oh

**Comment by peterhuene** [comment]
Also, can we export this type from the `durable` mod?

File: azure-functions/src/durable.rs [hunk 2]
```diff
@@ -12,6 +12,7 @@ use std::{
     task::{Context, Poll, RawWaker, RawWakerVTable, Waker},
 };
 
+mod activity_output;
```
Comment (peterhuene) [inline azure-functions/src/durable.rs]: We should export everything in that module into the current (`durable`) module with `pub use self::activity_output::*;` (add to the list of below `pub use` statements).

**Comment by t-eckert** [comment]
Any other tests you can think of?

**Comment by t-eckert** [comment]
I'm going to add more documentation before this is merged.

**Comment by peterhuene** [comment]
I think that's good for now.  This needs a rebase on `durable-functions` to resolve a conflict.

**Comment by t-eckert** [comment]
Should be ok

**Comment by peterhuene** [comment]
👍 merge when you're ready :)

**Pull request merged**